presto-main/src/test/java/io/prestosql/sql/query/QueryAssertions.java

```diff
@@ -347,11 +418,16 @@ public QueryAssert isNotFullyPushedDown(Class<? extends PlanNode> retainedNode)
         }
 
         private void verifyResultsWithPushdownDisabled()
+        {
+            verifyResultsWithPushdownDisabled(List.of());
```

Comment:
We lean towards ImmutableList still, see https://prestosql.slack.com/archives/CP1MUNEUX/p1596109740319900

Reply:
Done.